docs/en/reference/modules/EmailTemplates.md

```diff
@@ -0,0 +1,42 @@
+# EmailTemplates (`StatCan.OrchardCore.EmailTemplates`)
+
+The EmailTemplates module aims to simplify and accelerate the process of sending emails by using templates.
+
+## Content definitions
+
+## EmailTemplate
```

Comment:
I think you need to split this section in two, the EmailTemplates and the EmailTemplatePart